addons/base_vat/models/res_partner.py

```diff
@@ -79,7 +79,6 @@
     've': 'V-12345678-1, V123456781, V-12.345.678-1',
     'xi': 'XI123456782',
     'sa': _lt('310175397400003 [Fifteen digits, first and last digits should be "3"]'),
-    'jp': 'T7000012050002',
```

Comment:
@vin-odoo Not sure if we need to keep this number after looking at https://github.com/odoo/odoo/commit/cb9dc2b037992fd64d19cea012a6c9f92dde9f04 🤔

Reply:
The format is: `T+13-Digit Corporate Number`
So both the VAT and company registry were made to match indeed.

The actual number doesn't matter much, but for clarity it's best to keep them aligned.